rules/transition_support.bzl

```diff
@@ -7,6 +7,51 @@ load(
     "get_bazel_version",
 )
 
+_PLATFORM_TYPE_TO_CPUS_FLAG = {
+    "ios": "//command_line_option:ios_multi_cpus",
+    "macos": "//command_line_option:macos_cpus",
+    "tvos": "//command_line_option:tvos_cpus",
+    "visionos": "//command_line_option:visionos_cpus",
+    "watchos": "//command_line_option:watchos_cpus",
+}
+
+# Could not load it directly from bazel_build_apple_support
+# As we support multiple versions of rules_apple_support
+# Where some don't have the file. So we copied it
```

Comment:
Can you add a TODO to switch to directly loading it when the minimum apple_support version is X?

Reply:
Added ✅